tools/clang/test/HLSLFileCheck/shader_targets/library/vec_array_param.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: %dxc -T lib_6_6 %s | FileCheck %s
+
+// Make sure vector array param works.
```

Comment:
Could we have a check for the bitcast operator here?

Reply:
Sure.